assets/theme-css/pst/base/_base.scss

```diff
@@ -160,6 +160,11 @@ pre {
   border: 1px solid var(--pst-color-border);
   border-radius: $admonition-border-radius;
 
+  // Prevent long, monospaced lines and words in PRE elements from
+  // breaking page layout.  Very non-obvious problem and solution.
+  white-space: pre-wrap;
+  word-break: break-all;
+
```

Comment:
Could you add a comment to explain why this is the correct solution? That would be helpful to reviewers now, and to our future selves.